sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/object.scala

```diff
@@ -286,6 +290,9 @@ case class FlatMapGroupsInR(
     child: LogicalPlan) extends UnaryNode with ObjectProducer{
 
   override lazy val schema = outputSchema
+
+  override protected def stringArgs: Iterator[Any] = Iterator(inputSchema, outputSchema,
+    outputObjAttr, child)
```

Comment:
Can we add `groupingAttributes` and `dataAttributes` as well here ? They shouldn't contain R serialized data from what I can see.